lib/metanorma/standoc/latexml_requirement.rb

```diff
@@ -0,0 +1,43 @@
+require_relative "./requirement"
+
+module Metanorma
+  module Standoc
+    class LatexmlRequirement < Requirement
+      def initialize
+        @recommended_version = '0.8.4'
+        @minimal_version = '0.8.0'
+        version_output, = Open3.capture2e("latexml --VERSION")
+        @actual_version = version_output&.match(%r{\d+(.\d+)*})
+      end
+
+      def satisfied
+        version = @actual_version
+
+        if version.to_s.empty?
+          abort "LaTeXML not found in PATH, please make sure that you installed LaTeXML"
+        elsif Gem::Version.new(version) < Gem::Version.new(@minimal_version)
```

Comment:
The `if` block will have already terminated due to the `abort`, so you can just do:

```ruby
if version.to_s.empty?
  abort "LaTeXML not found in PATH, please make sure that you installed LaTeXML"
end

if Gem::Version.new(version) < Gem::Version.new(@minimal_version)
  abort ...
end

if Gem::Version.new(version) < Gem::Version.new(@recommended_version)
  ...
  @cmd = "latexmlmath --preload=amsmath -- -"
else
  @cmd = "latexmlmath --preload=amsmath --inputencoding=UTF-8 -- -"
end
```

Reply:
Done